app/views/charity_overview/index.html.erb

```diff
@@ -0,0 +1,67 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Charities</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Regno</th>
+      <th>Subno</th>
+      <th>Name</th>
+      <th>Orgtype</th>
+      <th>Gd</th>
+      <th>Aob</th>
+      <th>Aob defined</th>
+      <th>Nhs</th>
+      <th>Ha no</th>
+      <th>Corr</th>
+      <th>Add1</th>
+      <th>Add2</th>
+      <th>Add3</th>
+      <th>Add4</th>
+      <th>Add5</th>
+      <th>Postcode</th>
+      <th>Phone</th>
+      <th>Fax</th>
+      <th>Lon</th>
+      <th>Lat</th>
+      <th>Update at</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @charities.each do |charity| %>
+      <tr>
+        <td><%= charity.regno %></td>
+        <td><%= charity.subno %></td>
+        <td><%= charity.name %></td>
+        <td><%= charity.orgtype %></td>
```

Comment:
do we need a charity.desc in here too?

Reply:
that is probably sensible... I don't know how many people use the index.html.erb, but I can't think of a solid reason why not to add it

Reply:
but not a reason to prevent this all being merged and deployed ...